R/fit_process.R

```diff
@@ -113,11 +90,9 @@ spim_fit_process <- function(samples, parameters, data, control,
   ## restart information from spimalot parent objects (i.e trajectories, prior,
   ##   rt, etc.)
   list(
-    fit = list(samples = samples_thin, # note complicated naming change here
-               pmcmc = samples,
+    fit = list(samples = samples, # note complicated naming change here
```

Comment:
comment now out of date, this is nice and simple again

Reply:
Have edited the comment